app/services/place_hold_form/builder.rb

```diff
@@ -44,10 +44,24 @@ def parsed_symphony_response(symphony_call, *params)
     # barcode from a random item in a random call in the @call_list is used (local logic in Symphony dictates).
     #
     # Second: sort naturally by Call#volumetric.
+    #
+    # Third: If there happen to be non-voluemtrics along-side the volumetrics, reduce them to just 1.
+    #
+    # Fourth: Only pass along the volumetric calls that have unique Call#call_number
     def process_volumetric_calls
       filter_holdables if @call_list.count > 1
       @volumetric_calls = @call_list.dup if volumetric? && @call_list.count > 1
       volumetric_natural_sort
+      compact_non_volumetric_calls if @volumetric_calls.select { |c| c.volumetric.nil? }.count > 1
+      @volumetric_calls.uniq!(&:call_number)
+    end
+
+    def compact_non_volumetric_calls
+      process_these = @volumetric_calls.dup
+      @volumetric_calls = []
+      process_these.each_with_index do |c, i|
```

Comment:
a minor thing but per our conversation, renaming `c`  to be more descriptive would be helpful here

Reply:
rename `c` or `process_these`?

i use `c` through the class as shorthand for `call`... 🤔 

`process_these` was a var i just forgot to rename before PRing... maybe I should rename both?

Reply:
oh I think `process_these` is fine, sorry I just meant `c`. I mean if you want to rename the `process_these` as well, your call but it is clear to read to me as is :)

Reply:
okie doke, hope that improves readability

Reply:
looks great, thanks for the updates!